### PR TITLE
API library

### PR DESCRIPTION
**Pull request by nottrobin**
Here I've tried to create a CertificationAPI class which
tries to reflect in method and property names the shape of the
certification API.

This will hopefully make development easier and clearer down the line.

It also allowed me to simplify the existing view functions a fair bit.

QA
--

```
./run
```

Browse the site, check the bits that worked before still work.

### CODE REVIEW

File: webapp/api.py [hunk 1]
```diff
@@ -1,36 +1,113 @@
...
+        )
 
+        # Raise any HTTP errors
+        response.raise_for_status()
 
-def get(endpoint):
-    return api_session.get(f"{base_url}/{endpoint}")
+        return response
 
+    def certifiedmakes(
```
Comment (jpmartinspt) [inline webapp/api.py:29]: Are we not using underscores too match API method names?

File: webapp/app.py [hunk 2]
```diff
@@ -1,172 +1,148 @@
...
-        release_version = release["certified_release"]
-        release_name = (
-            "Ubuntu "
-            f"{release_version} "
-            f"{ '64 Bit' if release['architecture'] == 'amd64' else '32 Bit'}"
-        )
+
+    for model_release in model_releases:
+        ubuntu_version = model_release["certified_release"]
+        arch = ""
```
Comment (jpmartinspt) [inline webapp/app.py:57]: We could `arch = "32 Bit"` here and remove the `else`

**Review by jpmartinspt — APPROVED** [review]
Seems to work fine, left a two small comments feel free to change or not!

File: webapp/app.py (hunk 2 above)
Comment (nottrobin) [reply webapp/app.py:57]: Yeah I considered that, but I think this version, although it's 2 lines longer, is a bit more logically straightforward which I think is more important.

File: webapp/api.py (hunk 1 above)
Comment (nottrobin) [reply webapp/api.py:29]: Yes, I thought it should match the exact endpoint of the API as closely as possible.

**Pull request merged**